systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java

```diff
@@ -100,6 +100,15 @@ void testRackAware() {
         List<Event> events = kubeClient().listEvents(uid);
         assertThat(events, hasAllOfReasons(Scheduled, Pulled, Created, Started));
 
+        LOGGER.info("Waiting for reconciliation to happen. " +
+                "Giving some time to DNS/load balancer to propagate kafka address." +
+                "Sleeping for {}ms", Constants.RECONCILIATION_INTERVAL);
+        try {
+            Thread.sleep(Constants.RECONCILIATION_INTERVAL);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
Why is this in any way relevant only to this ST? Also, sleeping for the whole reconciliation interval could be waste of time. You should check periodically with some small backoff for example to make sure you don't waste time on it.

Reply:
Removed with LB and External clients usage.